WebRoot/css/zm.css

```diff
@@ -4431,7 +4431,7 @@ HTML>BODY .appt-selected .appt_allday_body
 
 .shortcutListMap DT {
 	display: inline-block;
-	text-align: right;
+	text-align: left;
```

Comment:
Yes we can move this one to be as a property, but just changed alignment here and there are almost 20+ style rules related to shortcut here that are not using properties. Ideally if we think these should be customisable, we should change all the rules in one go. 
Let me know your thoughts on this.